### PR TITLE
Add yamllint to GHA

### PR DESCRIPTION
**Pull request by gliptak**
Signed-off-by: Gábor Lipták <gliptak@gmail.com>

https://github.com/SigmaHQ/sigma/pull/221

### CODE REVIEW

File: tools/config/winlogbeat.yml [hunk 1]
```diff
@@ -203,8 +203,6 @@ fieldmappings:
     ContextInfo: winlog.event_data.ContextInfo
     # from here missing field at 20210706
     Accesses: winlog.event_data.Accesses
-    AccessList: winlog.event_data.AccessList
```
Comment (Neo23x0) [inline tools/config/winlogbeat.yml:206]: Why did you remove these field mappings?

File: tools/config/winlogbeat-modules-enabled.yml [hunk 2]
```diff
@@ -225,7 +225,6 @@ fieldmappings:
     Accesses: winlog.event_data.Accesses
     AccessList: winlog.event_data.AccessList
     AttributeValue: winlog.event_data.AttributeValue
-    AttributeValue: winlog.event_data.AttributeValue
```
Comment (Neo23x0) [inline tools/config/winlogbeat-modules-enabled.yml:228]: Why did you remove these field mappings?

File: tools/config/winlogbeat.yml (hunk 1 above)
Comment (gliptak) [reply tools/config/winlogbeat.yml:206]: Duplicate keys invalidate YAML

File: tools/config/winlogbeat-modules-enabled.yml (hunk 2 above)
Comment (gliptak) [reply tools/config/winlogbeat-modules-enabled.yml:228]: Duplicate keys invalidate YAML

File: tools/config/winlogbeat.yml (hunk 1 above)
Comment (frack113) [reply tools/config/winlogbeat.yml:206]: sorry my bad :confounded:

File: tools/config/winlogbeat-modules-enabled.yml (hunk 2 above)
Comment (Neo23x0) [reply tools/config/winlogbeat-modules-enabled.yml:228]: Okay, thanks
